### ISSUE TITLE
CONTRIBUTING.md: `codespell` command lists `styles`, which no longer exists

### ISSUE THREAD

**Issue by TokenStyle** [wemake-services/wemake-python-styleguide#3783]
i ran the spellcheck commands from `CONTRIBUTING.md`, and the `codespell` one lists a `styles` diirectory:

    codespell -w wemake_python_styleguide tests docs scripts styles *.md --ignore-words ./tests/whitelist.txt

There is no `styles/` in the repo. It was removed in #3132 (commit d646837c) along with the preset files it held. `CONTRIBUTING.md` dates to #1799 and not updated.

tbh nothing actually breaks, `codespell` ignores a path that doesn't exist without any warning and still exits 0, so the command works fine. Tho a new contributor (me for e.g) will get confused when looking in `styles/` in my opinion.

```bash
$ ls -d styles
ls: cannot access 'styles': No such file or directory

$ git log --oneline --diff-filter=D -- 'styles/*'
d646837c Drops `nitpick` support (#3132)

$ codespell styles; echo "exit=$?"
exit=0
```

I would suggest just dropping `styles` from the command. Happy to send a PR if that sounds right.

One thing I noticed while checking this, not part of the above: the `flake8-spellcheck` command in the same section

    flake8 --whitelist ./tests/whitelist.txt .

produces no output, because `setup.cfg` has `select = WPS, E999` and `flake8-spellcheck` reports `SC` codes. Running the same command with `--select=SC` does report `SC200` warnings. I'm not sure what the intended fix is here (adjust the documented command, change `select`, or drop the section), so I left it out — happy to open a separate issue if it's worth tracking.

**Comment by sobolevn** [comment]
pr is welcome :)

**Comment by TokenStyle** [comment]
oh, so fast, thx!

**Comment by sobolevn** [comment]
I propose removing this at all, we run codespell in `pre-commit` now :)
